### ISSUE TITLE
IP leakage when changing the server

### ISSUE THREAD

**Issue by salvq** [Joentje/nordvpn-proxy#72]
There is an IP leakage when changing the server (i.e. openvpn restarts due to server overload)

When you kill the service, your original IP is leakage until connection is established to new server

[https://github.com/Joentje/nordvpn-proxy/blob/dcc8d7f6dba126d220f4c830a7f1a783c08b69fa/app/ovpn/get-status-server.sh#L18](https://github.com/Joentje/nordvpn-proxy/blob/dcc8d7f6dba126d220f4c830a7f1a783c08b69fa/app/ovpn/get-status-server.sh#L18)

Does the docker contain killswitch option or any plans to have it ?

Thanks,

**Comment by Joentje** [comment]
Any suggestions? When using Docker networking, and there is no vpn connection, does it still work for you?

**Comment by salvq** [comment]
Yes, when you kill the connection to VPN server, you are connected to internet without VPN.

One suggestion would be to figure out iptables and ip route parameters but I do not know all the details yet, just started learning about that.

Another option is to use nordvpn application for linux which provide feature called killswitch and will do the ip tables for you. However there is an issue to access the docker network from host from host, so also would requires iptables / route parameters adjustment.

**Comment by strictboss** [comment]
Any improvements for the issue? Really nice project idea. I like that, but killswitch is needed!

**Comment by strictboss** [comment]
> Yes, when you kill the connection to VPN server, you are connected to internet without VPN.
> 
> One suggestion would be to figure out iptables and ip route parameters but I do not know all the details yet, just started learning about that.
> 
> Another option is to use nordvpn application for linux which provide feature called killswitch and will do the ip tables for you. However there is an issue to access the docker network from host from host, so also would requires iptables / route parameters adjustment.

@salvq Maybe you really can improve the project in the context of working killswitch?
